demosplan/DemosPlanCoreBundle/Logic/Import/Statement/StatementFromRowBuilderWithZipSupport.php

```diff
@@ -19,6 +26,10 @@
 
 class StatementFromRowBuilderWithZipSupport extends AbstractStatementFromRowBuilder
 {
+    protected Cell $fileReferences;
+
+    protected Cell $originalFileReferences;
```

Comment:
I think this can be `null`. Thus, `null` needs to be handled when accessing this property.

Reply:
The accesses to these properties still need to handle `null` gracefully.

Reply:
done in [5018d06](https://github.com/demos-europe/demosplan-core/pull/2575/commits/5018d06ad508b8dcd5f0812c9e3319b038ee892c)